lib/engine.py

```diff
@@ -348,17 +350,18 @@ def create_table(self):
             print("Couldn't create table (%s). Trying to continue anyway." % e)
 
     def create_table_statement(self):
-        """Returns a SQL statement to create a table."""
-        create_stmt = "CREATE TABLE " + self.table_name() + " ("
-
-        columns = self.table.get_insert_columns(join=False)
+        """Returns a SQL statement to create a table
 
+        use create=True to get insert columns including Auto field for creating table
+        default is false and used when returning only the colums for inserts 
```

Comment:
It looks like these new lines in the docstring should be in `get_insert_columns` instead of `create_table_statement`.